docs/sample_config.yaml

```diff
@@ -1525,47 +1528,52 @@ trusted_key_servers:
 # https://<server>:<port>/_matrix/saml2/authn_response.
 #
 saml2_config:
+  # Uncomment the following to enable authorization against a SAML server.
+  # Defaults to true, but is unused unless sp_config exists.
+  #
+  #enabled: true
```

Comment:
this was omitted because it seemed to be redundant.

Reply:
Yeah it is a little confusing. OIDC did the reasonable thing and defaults this to false unless it exists, so was trying to make it consistent. I can remove it if you'd like.

Reply:
> OIDC did the reasonable thing and defaults this to false unless it exists, so was trying to make it consistent

In the past, I've waged a bit of a war on `enabled` options: my feeling is that you should just omit the section if you don't want that functionality. So I'm not convinced that *is* the reasonable thing, and hence leaving the `enabled` setting undocumented here.

In short, I'd vote for leaving it undocumented.

Reply:
"Reasonable" meaning that *if* it exists it should default to disabled. I'm OK removing this section though.